main.py

```diff
@@ -342,7 +343,7 @@ def missing_page(campaign_group=None):
             ) for r in rows]
     rows = sort_rows(rows, 5)
     rows = add_counters(rows)
-    aggregate_rows(rows, 5)
+    #aggregate_rows(rows, 5)
```

Comment:
If you remove `aggregate_rows`, you can also remove `add_counters` as well, because counters are needed only for aggregation, i.e. merging of rows. If you remove counters, then HTML will change as well, as now each cell will be a single value instead of a tuple. This means that all `row[X][0]` are now `row[X]` and all ` {% if row[X][1] > 0 %}` can be removed all together.